host_cleanup.sh

```diff
@@ -7,7 +7,7 @@ source lib/logging.sh
 source lib/common.sh
 
 # Kill and remove the running ironic containers
-for name in ipa-downloader ironic ironic-inspector ironic-endpoint-keepalived dnsmasq httpd mariadb vbmc sushy-tools httpd-infra; do
+for name in ipa-downloader ironic ironic-api ironic-conductor ironic-inspector ironic-endpoint-keepalived dnsmasq httpd mariadb vbmc sushy-tools httpd-infra; do
```

Comment:
Sorry if I'm missed any description for this container, but why do we need this container? and what does it do different compared to ironic one?

Reply:
it splits the two ironic processes in their own containers, api and conductor. That's better practice.